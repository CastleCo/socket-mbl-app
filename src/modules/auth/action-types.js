export const SET_USER = '@@AUTH/SET_USER';
export const SET_TOKENS = '@@AUTH/SET_TOKENS';

export const LOGIN = {
  REQUESTED: '@@AUTH/LOGIN/REQUESTED',
  SUCCEEDED: '@@AUTH/LOGIN/SUCCEEDED',
  FAILED: '@@AUTH/LOGIN/FAILED',
};

export const REGISTER = {
  REQUESTED: '@@AUTH/REGISTER/REQUESTED',
  SUCCEEDED: '@@AUTH/REGISTER/SUCCEEDED',
  FAILED: '@@AUTH/REGISTER/FAILED',
};

export const LOGOUT = {
  REQUESTED: '@@AUTH/LOGOUT/REQUESTED',
  SUCCEEDED: '@@AUTH/LOGOUT/SUCCEEDED',
  FAILED: '@@AUTH/LOGOUT/FAILED',
};