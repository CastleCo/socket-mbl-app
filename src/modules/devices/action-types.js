
export const REPLACE_DEVICE_LIST = 'DEVICES/SET_DEVICE_LIST';
export const EXTEND_DEVICE_LIST = 'DEVICES/EXTEND_DEVICE_LIST';
export const REFRESH_DEVICES = 'DEVICES/REFRESH_DEVICES';
export const REQUEST_MORE_DEVICES = 'DEVICES/REQUEST_DEVICE_LIST';
export const REQUEST_MORE_DEVICES_FAILED = 'DEVICES/REQUEST_DEVICE_LIST_FAILED';