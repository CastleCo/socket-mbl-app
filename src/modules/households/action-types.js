
//// REQUESTING HOUSEHOLDS

export const REPLACE_HOUSEHOLD_LIST = `HOUSEHOLDS/SET_HOUSEHOLD_LIST`;
export const REQUEST_MORE_HOUSEHOLDS = `HOUSEHOLDS/REQUEST_MORE_HOUSEHOLDS`;
export const REQUEST_MORE_HOUSEHOLDS_FAILED = `HOUSEHOLDS/REQUEST_MORE_HOUSEHOLDS_FAILED`;

export const REFRESH_HOUSEHOLDS = `HOUSEHOLDS/REFRESH_HOUSEHOLDS`;
export const REFRESHHOUSEHOLDS_FAILED = `HOUSEHOLDS/REFRESH_HOUSEHOLDS_FAILED`;

//// REQUESTING A HOUSEHOLD

export const REQUEST_HOUSEHOLD = `HOUSEHOLDS/REQUEST_HOUSEHOLD`;
export const REQUEST_HOUSEHOLD_FAILED = `HOUSEHOLDS/REQUEST_HOUSEHOLD_FAILED`;

//// SETTING THE ACTIVE HOUSEHOLD

export const REQUEST_ACTIVE_HOUSEHOLD = `HOUSEHOLDS/REQUEST_ACTIVE_HOUSEHOLD`;
export const REQUEST_ACTIVE_HOUSEHOLD_FAILED = `HOUSEHOLDS/REQUEST_ACTIVE_HOUSEHOLD_FAILED`;
export const SET_ACTIVE_HOUSEHOLD = `HOUSEHOLDS/SET_ACTIVE_HOUSEHOLD`