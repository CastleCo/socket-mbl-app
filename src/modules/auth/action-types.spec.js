import * as actionsTypes from "./action-types";

describe("auth.actionTypes", () => {
  it("should match the snapshot", () => {
    expect(actionsTypes).toMatchSnapshot();
  });
});