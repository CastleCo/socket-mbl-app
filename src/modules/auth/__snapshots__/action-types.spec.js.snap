// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`auth.actionTypes should match the snapshot 1`] = `
Object {
  "LOGIN": Object {
    "FAILED": "@@AUTH/LOGIN/FAILED",
    "REQUESTED": "@@AUTH/LOGIN/REQUESTED",
    "SUCCEEDED": "@@AUTH/LOGIN/SUCCEEDED",
  },
  "LOGOUT": Object {
    "FAILED": "@@AUTH/LOGOUT/FAILED",
    "REQUESTED": "@@AUTH/LOGOUT/REQUESTED",
    "SUCCEEDED": "@@AUTH/LOGOUT/SUCCEEDED",
  },
  "REGISTER": Object {
    "FAILED": "@@AUTH/REGISTER/FAILED",
    "REQUESTED": "@@AUTH/REGISTER/REQUESTED",
    "SUCCEEDED": "@@AUTH/REGISTER/SUCCEEDED",
  },
  "SET_TOKENS": "@@AUTH/SET_TOKENS",
  "SET_USER": "@@AUTH/SET_USER",
}
`;
